<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node name="yamaha_rosbridge_client_node" pkg="yamaha_rosbridge_client" type="rosbridge_connector.py" 
          output="screen">
        <param name="primary_host" value="192.168.127.103"/>
        <param name="primary_port" value="9090"/>
        <param name="redundant_host" value="192.168.127.104"/>
        <param name="redundant_port" value="9090"/>
        <param name="reconnection_period" value="0.3"/>
    </node>
</launch>
